test/unit/webdriver/device/keyboard_test.py

```diff
@@ -32,3 +32,27 @@ def test_hide_keyboard(self):
             appium_command('/session/1234567890/appium/device/hide_keyboard')
         )
         assert isinstance(driver.hide_keyboard(), WebDriver)
+
+    @httpretty.activate
+    def test_press_keycode(self):
+        driver = android_w3c_driver()
+        httpretty.register_uri(
+            httpretty.POST,
+            appium_command('/session/1234567890/appium/device/press_keycode'),
+            body='{"value": "86"}'
+        )
+        driver.press_keycode('86')
```

Comment:
Check the return value.
And arg is `int`.

https://github.com/appium/python-client/blob/9e18dfce1ccbe34a5bba3684b054e7f5e22bd229/appium/webdriver/extensions/keyboard.py#L79